testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/ConfigurationTools.cs

```diff
@@ -0,0 +1,92 @@
+using UnityEngine;
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Text;
+
+namespace Unity.Netcode.MultiprocessRuntimeTests
+{
+    public class ConfigurationTools
+    {
+        public static JobQueueItemArray GetRemoteConfig()
+        {
+            var theList = new JobQueueItemArray();
+            try
+            {
+                using var client = new HttpClient();
+                var cancelAfterDelay = new CancellationTokenSource(TimeSpan.FromSeconds(10));
+                var responseTask = client.GetAsync("https://multiprocess-log-event-manager.cds.internal.unity3d.com/api/JobWithFile",
+                    HttpCompletionOption.ResponseHeadersRead, cancelAfterDelay.Token);
+                responseTask.Wait();
```

Comment:
If we have devs running those tests locally at the same time, could there be a possibility of conflict with that shared ressource?

Reply:
same githash == two devs both running their tests from develop for example?
ok

Reply:
I will think of another solution for the local case.